src/lib.rs

```diff
@@ -366,9 +366,9 @@ impl Default for SecretKey {
     }
 }
 
-impl Into<Scalar> for SecretKey {
```

Comment:
Why is `Into<Scalar>` removed?

Reply:
Because this creates an error.
You can't move out the inside of a type that implements Drop

Reply:
Maybe this will work:

```
let ret = self.0.clone();
ret
```

I understand that this may be a little bit more inefficient but it preserves the API, as someone might need it.

Reply:
Hmm ok, although that will leak the scalar and then the Drop won't run on it

Reply:
Done